indico/modules/events/management/controllers/settings.py

```diff
@@ -139,6 +142,19 @@ class RHEditEventClassification(RHEditEventDataBase):
     form_class = EventClassificationForm
     section_name = 'classification'
 
+    def _process(self):
+        allowed_keywords = misc_settings.get('allowed_keywords')
+        if allowed_keywords:
+            default = list(filter(lambda kw: kw in allowed_keywords, self.event.keywords))
```

Comment:
Use a list comprehension for this. Actually, you can probably even use set operations here

Reply:
Changed.